modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/UserResourceManagerTest.java

```diff
@@ -95,15 +346,19 @@ public void testGetSuccess(String id, String attributes,
                 (SCIMResourceTypeSchema) CopyUtil.deepCopy(schema), attributes, excludeAttributes);
 
         abstractResourceManager.when(() -> AbstractResourceManager.getResourceEndpointURL(SCIMConstants.USER_ENDPOINT))
-                .thenReturn(SCIM2_ME_ENDPOINT);
+                .thenReturn(SCIM2_USER_ENDPOINT);
 
-        abstractResourceManager.when(() -> userManager.getUser(id, requiredAttributes)).thenReturn(user);
+        Mockito.when(userManager.getUser(id, requiredAttributes)).thenReturn(user);
 
-        SCIMResponse outputScimResponse = userResourceManager.get(id, userManager, attributes, excludeAttributes);
-        JSONObject obj = new JSONObject(outputScimResponse.getResponseMessage());
+        SCIMResponse scimResponse = userResourceManager.get(id, userManager, attributes, excludeAttributes);
+        JSONObject obj = new JSONObject(scimResponse.getResponseMessage());
 
-        Assert.assertEquals(outputScimResponse.getResponseStatus(), expectedScimResponseStatus);
+        Assert.assertEquals(scimResponse.getResponseStatus(), ResponseCodeConstants.CODE_OK);
         Assert.assertEquals(obj.getString("id"), id);
+
+        String returnedURI = scimResponse.getHeaderParamMap().get(SCIMConstants.LOCATION_HEADER);
+        String expectedURI = SCIM2_USER_ENDPOINT + "/" + obj.getString("id");
+        Assert.assertEquals(returnedURI, expectedURI);
```

Comment:
better to perform some assertions based on the values passed for `attributes` and `excludeAttributes` from the dataholder.

Reply:
Added commit 6f5ca461fa07ac4b49182320325156a58b80b910